pyvista/plotting/plotter.py

```diff
@@ -1281,69 +1284,73 @@ def disable_anti_aliasing(self, all_renderers=True):
             self.renderer.disable_anti_aliasing()
 
     @wraps(Renderer.set_focus)
-    def set_focus(self, *args, render=True, **kwargs):  # numpydoc ignore=PR01,RT01
+    def set_focus(self, *args, render=True, **kwargs) -> None:  # numpydoc ignore=PR01,RT01
         """Wrap ``Renderer.set_focus``."""
         log.debug('set_focus: %s, %s', str(args), str(kwargs))
         self.renderer.set_focus(*args, **kwargs)
         if render:
             self.render()
 
     @wraps(Renderer.set_position)
-    def set_position(self, *args, render=True, **kwargs):  # numpydoc ignore=PR01,RT01
+    def set_position(self, *args, render=True, **kwargs) -> None:  # numpydoc ignore=PR01,RT01
         """Wrap ``Renderer.set_position``."""
         self.renderer.set_position(*args, **kwargs)
         if render:
             self.render()
 
     @wraps(Renderer.set_viewup)
-    def set_viewup(self, *args, render=True, **kwargs):  # numpydoc ignore=PR01,RT01
+    def set_viewup(self, *args, render=True, **kwargs) -> None:  # numpydoc ignore=PR01,RT01
         """Wrap ``Renderer.set_viewup``."""
         self.renderer.set_viewup(*args, **kwargs)
         if render:
             self.render()
 
     @wraps(Renderer.add_orientation_widget)
-    def add_orientation_widget(self, *args, **kwargs):  # numpydoc ignore=PR01,RT01
+    def add_orientation_widget(
+        self, *args, **kwargs
+    ) -> _vtk.vtkOrientationMarkerWidget:  # numpydoc ignore=PR01,RT01
         """Wrap ``Renderer.add_orientation_widget``."""
         return self.renderer.add_orientation_widget(*args, **kwargs)
 
     @wraps(Renderer.add_axes)
-    def add_axes(self, *args, **kwargs):  # numpydoc ignore=PR01,RT01
+    def add_axes(self, *args, **kwargs) -> AxesActor:  # numpydoc ignore=PR01,RT01
```

Comment:
I also don't think is quite right, the only place `pv.AxesActor` is returned in the code base is with `pv.Axes.axes_actor`. I believe `vtk.vtkAxesActor` should be the return type here.

Reply:
Taking a look at Renderer's `add_axes`, `self.axes_actor` is either created using `create_axes_orientation_box` (which seems to return either `_vtk.vtkPropAssembly` or `_vtk.vtkAnnotatedCubeActor`) or using `create_axes_marker` (which returns `_vtk.vtkAxesActor`).

So I put `_vtk.vtkAxesActor | _vtk.vtkPropAssembly | _vtk.vtkAnnotatedCubeActor`